modules/byo-vpc-single-az-ocm-config.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_create/managing-byo-vpc-single-az.adoc
+
+[id="byo-vpc-single-az-ocm-config_{context}"]
+= Configuring your VPC with a single availability zone in OCM
+
+Once the subnets are properly configured with the necessary routing rules and NAT gateway, the final configuration steps are done in OCM.
+
+.Procedure
+
+. Log in to {cloud-redhat-com}.
+. Click *Create Cluster* -> *Red Hat OpenShift Dedicated* -> *Create Clusters*.
+. Select *Customer Cloud Subscription (CCS)* for the billing model.
+. Select *AWS* as the cloud infrastructure provider.
+. Enter the AWS Account ID for the `osdCcsAdmin` IAM User on the *Cluster Creation* page.
+. Select *Single zone* under *Availability*.
+. Select *Advanced* for networking configuration.
```

Comment:
Similar to my previous comment, consider swapping the order of these for usability.